pkg/client/cli/telepresence_test.go

```diff
@@ -399,6 +395,14 @@ func (cs *connectedSuite) SetupSuite() {
 	require := cs.Require()
 	c := dlog.NewTestContext(cs.T(), false)
 
+	// Connect + quit before we change contexts to ensure the
+	// traffic-manager is installed
+	_, stderr := telepresence(cs.T(), "connect")
+	require.Empty(stderr)
+	time.Sleep(time.Second) // Allow some time before we quit
+	_, stderr = telepresence(cs.T(), "quit")
+	require.Empty(stderr)
+
```

Comment:
If you rebase this on `release/v2` then there's a capturePodLogs() that should be moved to this location.